lib/activesupport/all/activesupport_duration_test.rb

```diff
@@ -62,4 +62,9 @@ module ActiveSupportDurationTest
 
   T.assert_type!(3.years % 5, ActiveSupport::Duration)
   T.assert_type!(3.years % 5.days, ActiveSupport::Duration)
+
+  # see https://github.com/sorbet/sorbet-typed/pull/190
+  T.assert_type!(Date.today - 4.weeks, Date)
+  T.assert_type!(Date.today - Date.today, Rational)
```

Comment:
this test requires a relatively recent version of sorbet to pass (it should match https://github.com/sorbet/sorbet/pull/2567). i'm not sure how CI works with that - i guess we're about to find out.